resources/dicts/thoughts/alive/general.json

```diff
@@ -224,7 +264,10 @@
         "thoughts": [
             "Is wondering what r_c is thinking",
             "Is hoping r_c notices {PRONOUN/m_c/poss} improvement",
-            "Is in awe of r_c's leadership"
+            "Is in awe of r_c's leadership",
+            "Thinks r_c could be leading the Clan better than {PRONOUN/r_c/subj} {VERB/r_c/are/is}",
```

Comment:
Incorrect pronoun tags - the tag should be {PRONOUN/r_c/subject}. Alternatively, the thought might be snappier with out the last pronouns. IE., "Thinks r_c would be leading the clan better"